python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -284,18 +289,22 @@ def base_resource_usage(self) -> ExecutionResources:
         )
 
     def current_processor_usage(self) -> ExecutionResources:
-        # Both pending and running actors count towards our current resource usage.
-        num_active_workers = self._actor_pool.current_size()
+        # Only alive actors count towards our current resource usage.
+        num_alive_workers = self._actor_pool.num_alive_actors()
```

Comment:
my bad. all actors should count towards this. 
ResourceManager depends on this for resource budgeting.

Reply:
Ah ok. I will revert this.